packages/libp2p/src/circuit-relay/transport/index.ts

```diff
@@ -380,8 +382,10 @@ class CircuitRelayTransport implements Transport {
       localAddr
     })
 
-    log('new inbound connection %s', maConn.remoteAddr)
-    await this.upgrader.upgradeInbound(maConn)
+    log('new inbound transient connection %s', maConn.remoteAddr)
```

Comment:
we have a specific debug formatter for multiaddrs

```suggestion
    log('new inbound transient connection %a', maConn.remoteAddr)
```

Reply:
Done